dashboard/app/public_json_api.go

```diff
@@ -26,8 +27,9 @@ type PublicAPICrashDescription struct {
 func GetExtAPIDescrForBugPage(bugPage *uiBugPage) *PublicAPIBugDescription {
 	crash := bugPage.Crashes.Crashes[0]
 	return &PublicAPIBugDescription{
-		Version: 1,
-		Title:   bugPage.Bug.Title,
+		Version:        1,
+		Title:          bugPage.Bug.Title,
+		DiscussionLink: bugPage.Bug.ExternalLink,
```

Comment:
Given that we also collect discussions and patches from LKML (unlike ExternalLink, which points to Google groups), won't it happen that at some later point we'll want to also share that list via this API?

Reply:
Updated.